cityofaustin/atd-vz-data
# Update the atd-vzd readme

## Associated issues

This updates the atd-vzd readme to reflect our current Hasura migration and metadata update steps. I added couple of files to help us do the Hasura CLI stuff.

## Testing
**URL to test:** 
local

**Steps to test:**

Y'all can run through the steps if you want, but I think this one is mostly reading through the steps to see if I made sense and catching typos.

---
#### Ship list
